### DISCUSSION TITLE
Azure Pipeline publishing in a "S" project everytime

### DISCUSSION THREAD

**Discussion by ViniOkamoto** [vercel/community#4338] [Help] [answered]
### Page to Investigate

https://vercel.com/docs/deployments/git/vercel-for-azure-pipelines.

### Steps to Reproduce

Configure a Pipeline in Azure DevOps by following the documentation step by step below:
https://vercel.com/docs/deployments/git/vercel-for-azure-pipelines.

![image](https://github.com/vercel/community/assets/53801461/70fcda91-4df1-460a-9993-07e0233cac55)

The problem is happening when the pipeline runs. All the configs seem to work, and the code runs perfectly well. However, when I go to check if the project is deployed in the right project in Vercel, it's possible to see another project created with the letter "S" instead of deploying in the right one. 

![Screenshot 2023-10-03 at 12 43 33 AM](https://github.com/vercel/community/assets/53801461/2cd61601-8afb-4062-a900-577a0905f172)

Does anyone is facing the same problem?

**Comment by amyegan** [comment]
Hey @ViniOkamoto. I want to be sure I understand the situation correctly. Are you saying that your pipeline works to deploy your project and _also_ creates an additional "S" project? Or is it only deploying to the "S" project?

**Reply by ViniOkamoto** [reply]
It's only deploying on the "S" project. The weird thing is that I didn't create the project with the "S" name.

**Reply by ViniOkamoto** [reply]
I checked the environment variable to ensure I targeted the right project ID, but I got the same problem.

**Reply by ViniOkamoto** [reply]
Hello, @amyegan. Do you have any clues on why it might be happening?

**Reply by johanbove** [reply]
I am seeing the same behaviour. When the Azure pipeline runs, it all seems to be going well, but within the logs we can see that the mysterious "s" project is created instead of using the expected one. I am getting the same problem without using "vercel-deployment-task@1".

**Reply by johanbove** [reply, marked as answer]
Figured out this is probably related to this: <https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checkout-path>

**Reply by amyegan** [reply]
Thanks for sharing that! I checked with the team and confirmed that the default root directory does seem to be the cause. 

Azure pipelines always create a root directory named 's'. If you still see this after checking that you have set the Project and Org ID appropriately for the pipeline, then you may also need to add the `rootDirectory: ` option to the inputs for the deploy step as well to stop Azure from using the created `s` directory as the root by default.
